calculator.js

```diff
@@ -0,0 +1,57 @@
+/*
+<button> elements for the buttons, <button>7</button>
+<input> element for the display
+
+.value changes contents of the display
+*/
+
+/*global math*/
+
+var div = document.createElement('div');
+div.id = 'calculator';
+document.body.appendChild(div);
+
+var inputArea = document.createElement('input');
+inputArea.id = 'inputArea';
+div.appendChild(inputArea);
+
+
+
+var buttons = ['7','8','9','+','4','5','6','-','1','2','3','*','0','.','C','/','(',')','Del','='];
+
+buttons.forEach(makeButton);
+
+function makeButton(text) {
+    var button = document.createElement('button');
+    button.id = 'button' + text;
+    button.textContent = text;
+    div.appendChild(button);
+    button.addEventListener("click", buttonPress, false);
+}
+
+function buttonPress(event) {
+    console.log(event.target.innerHTML);
+    switch (event.target.innerHTML) {
```

Comment:
It would be slightly better to use the `.textContent` property instead of `.innerHTML`.

I would also recommend storing commonly-used values in variables, and refer to them in the rest of your function.

```js
var buttonValue = event.target.innerHTML;
var inputArea = document.getElementById('inputArea');
```

That will clean up this function a lot.